backend/api/student/simulation_instances.py

```
@@ -3,41 +3,281 @@
+                        # Create the instance with unique ID
+                        instance = StudentSimulationInstance(
+                            unique_id=generate_unique_simulation_instance_id(db),
+                            cohort_assignment_id=cohort_simulation.id,
+                            student_id=current_user.id,
+                            user_progress_id=user_progress.id
+                        )
+                        db.add(instance)
+                        db.commit()
+                        db.refresh(instance)
+                        logger.info(f"Auto-created simulation instance {instance.unique_id} for student {current_user.id}, cohort_simulation {cohort_simulation.id}")
+                    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)